photogrammetry_importer/utility/blender_point_utility.py

```diff
@@ -30,10 +35,10 @@ def compute_particle_coord_texture(coords, name='ParticleCoord'):
         alpha=False,
         # is_data=True,
         float_buffer=True,
-        width=len(coords), 
-        height=1)
+        width=MAX_WIDTH if len(coords) > MAX_WIDTH else len(coords), 
+        height=math.ceil(len(coords)/MAX_WIDTH))
 
-    copy_values_to_image(colors, image.name)
+    copy_values_to_image(coords, image.name)
```

Comment:
I'm not sure. It looked like a copy-paste error since I don't think colors is defined in that function and it seemed like the intent was for `compute_particle_coord_texture` to do the same thing as `compute_particle_color_texture` but for float values.
The change seemed like it fit with the intent of the function. Probably should have been in its own PR.